python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -159,6 +159,9 @@ def __init__(self, step_execution_context: StepExecutionContext):
         self._events: List[DagsterEvent] = []
         self._output_metadata: Dict[str, Any] = {}
 
+        self._requires_typed_event_stream = False
```

Comment:
TODO - figure out why this diff is still showing

Reply:
hm? Looks like this is moving this from the step context to be directly on the op context. I assume this was to make something work better with how the class hierarchy is set-up. If we dont need to make the change we shouldn't but I think its fine if the tests pass

Reply:
for context, i was looking at moving these attrs to the op context to make some upstack changes potentially easier. so i made a PR https://github.com/dagster-io/dagster/pull/18521 but the changes are still showing in this PR event though it's stacked on https://github.com/dagster-io/dagster/pull/18521

I think i'm going to close https://github.com/dagster-io/dagster/pull/18521 though since it's not strictly required right now